src/core/folder.h

```diff
@@ -179,6 +183,8 @@ private Q_SLOTS:
     bool defer_content_test : 1;
 
     static std::unordered_map<FilePath, std::weak_ptr<Folder>, FilePathHash> cache_;
+    static FilePath cutFilesDirPath_;
+    static std::shared_ptr<const HashSet> cutFilesHashSet_;
```

Comment:
I think `cutFilesHashSet_` shouldn't be static, otherwise it'll cause the problem mentioned at https://github.com/lxde/libfm-qt/pull/88#issuecomment-319429259.

Moreover, as far as I see. `cutFilesDirPath_` is redundant and `setCutFiles()` doesn't need a `dirPath` argument because it should always be `dirPath_`.

I made the above changes (offline) and https://github.com/lxde/libfm-qt/pull/88#issuecomment-319429259 was fixed. Am I missing something here?